torch/ao/nn/quantized/modules/activation.py

```diff
@@ -68,6 +68,23 @@ def forward(self, input):
     def _get_name(self):
         return 'QuantizedHardswish'
 
+    # ===== Serialization methods =====
```

Comment:
thanks for the fix!  To be consistent with other similar modules, could we do this by saving scale and zero_point as buffers intead?  https://github.com/pytorch/pytorch/blob/f172c7c60a5890136ea5f6225a56e41cb5d6e38f/torch/ao/nn/quantized/modules/normalization.py#L21 is an example.

Reply:
@vkuzo Changed and thanks for the suggestions. Please help to take a look again.